nservicebus/operations/tuning.md

```diff
@@ -31,6 +31,8 @@ You can define a maximum value for the number of messages per second that the en
 NServiceBus will not enforce any throughput restrictions by default.
 
 
+NOTE: Currently version 6 of NServiceBus does not support `MaximumMessageThroughputPerSecond` setting.
```

Comment:
since this is the new standard moving forward shouldn't this be at the top of this section?